### ISSUE TITLE
Keyboard Help Dialogue shows indent lines in a screenshot from the screenshot wrapper 

### ISSUE THREAD

**Issue by phet-steele** [phetsims/balloons-and-static-electricity#245]
Presumably not a big problem since it seems like an ultra rare use case. Taking a chance this is a sim bug, and not a PhET-iO bug, so reporting it in this repo. However, I am unaware how you would get a screenshot of this dialogue without PhET-iO, so adding that label:

<img width="710" alt="screen shot 2017-02-08 at 12 17 26 pm" src="https://cloud.githubusercontent.com/assets/5863899/22753467/6964d636-edf9-11e6-8c3e-ae100d554998.png">

Seen on macOS 10.12.3 Chrome at this link https://phet-io.colorado.edu/sims/balloons-and-static-electricity/1.2.0-phetio/wrappers/screenshot/screenshot.html?sim=balloons-and-static-electricity

**Comment by samreid** [comment]
@jessegreenberg how are those indents rendered?

**Comment by jessegreenberg** [comment]
Oops, sorry, with an HSeparator instead of a Spacer.

**Comment by jessegreenberg** [comment]
@samreid  this should be fixed, here is a screenshot taken with the wrapper with the above commit:

![capture](https://cloud.githubusercontent.com/assets/6396244/22905124/4ed4f092-f1fc-11e6-9f0f-bd80c5a59370.PNG)

Closing.
